shared/i18n/locales/en_US/translation.json

```diff
@@ -432,7 +432,8 @@
   "Nested documents are not shared": "Nested documents are not shared",
   "{{ teamName }} is using Outline to share documents, please login to continue.": "{{ teamName }} is using Outline to share documents, please login to continue.",
   "Are you sure you want to delete the <em>{{ documentTitle }}</em> template?": "Are you sure you want to delete the <em>{{ documentTitle }}</em> template?",
-  "Are you sure about that? Deleting the <em>{{ documentTitle }}</em> document will delete all of its history and any nested documents.": "Are you sure about that? Deleting the <em>{{ documentTitle }}</em> document will delete all of its history and any nested documents.",
+  "Are you sure about that? Deleting the <em>{{ documentTitle }}</em> document will delete all of its history and <em>{{ count }} nested documents</em>._{undefined}": "Are you sure about that? Deleting the <em>{{ documentTitle }}</em> document will delete all of its history and <em>{{ count }} nested documents</em>._{undefined}",
+  "Are you sure about that? Deleting the <em>{{ documentTitle }}</em> document will delete all of its history and <em>{{ count }} nested documents</em>._{undefined}_plural": "Are you sure about that? Deleting the <em>{{ documentTitle }}</em> document will delete all of its history and <em>{{ count }} nested documents</em>._{undefined}",
```

Comment:
Oops! These aren't the expected translations :thinking: I had added different ones. These are broken. I guess these got modified by the `precommit` hook. How can we fix these? Any pointers here? @tommoor

Reply:
Yea, the translation keys are generated automatically. The only time you need to edit this file is for the plural string – I'm not sure why `undefined` is here…

Reply:
Because of [this](https://github.com/outline/outline/pull/3876/files#diff-6fd6f22c7fed61dbbebc00cac6149e16260c7416cceae777c64b19095128a1dfR103). I was trying to set a [context](https://www.i18next.com/translation-function/context) for the zero nested docs scenario.

Reply:
It may be easier to split into two separate translations in the React tree based on the way this is behaving

Reply:
Yeah! Can't think of a better approach.

Reply:
Fixed!